tsconfig.json

```diff
@@ -2,6 +2,8 @@
 	"extends": "@sindresorhus/tsconfig",
 	"compilerOptions": {
 		"noEmit": true,
+		"skipLibCheck": false,
+		"strictNullChecks": true,
```

Comment:
what is the reason for adding this options?

Reply:
To make type inference of `null!` to be `never`, so that the `declare const nerver: never; expectType<...>(never)` can be simplified as `expectType<...>(null!)`; 
But for now, I think it's better to add a declaration of `never`(not to change `tsconfig.json`)